.circleci/config.yml

```diff
@@ -0,0 +1,252 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@2.0.3
+
+jobs:
+  build_doc:
+    executor: python/default
+    steps:
+      - restore_cache:
+          name: Restore .git
+          keys:
+            - source-cache-graphs
+      - checkout
+      - run:
+          name: Complete checkout
+          command: |
+            if ! git remote -v | grep upstream; then
+              git remote add upstream https://github.com/py-why/pywhy-stats.git
+            fi
+            git remote set-url upstream https://github.com/py-why/pywhy-stats.git
+            git fetch upstream
+      - save_cache:
+          name: Save .git
+          key: source-cache-graphs
+          paths:
+            - ".git"
+      - run:
+          name: Check-skip
+          command: |
+            set -e
+            export COMMIT_MESSAGE=$(git log --format=oneline -n 1);
+            if [[ -v CIRCLE_PULL_REQUEST ]] && ([[ "$COMMIT_MESSAGE" == *"[skip circle]"* ]] || [[ "$COMMIT_MESSAGE" == *"[circle skip]"* ]]); then
+              echo "Skip detected, exiting job ${CIRCLE_JOB} for PR ${CIRCLE_PULL_REQUEST}."
+              circleci-agent step halt;
+            fi
+      - run:
+          name: Merge with upstream
+          command: |
+            echo $(git log -1 --pretty=%B) | tee gitlog.txt
+            echo ${CI_PULL_REQUEST//*pull\//} | tee merge.txt
+            if [[ $(cat merge.txt) != "" ]]; then
+              echo "Merging $(cat merge.txt)";
+              git pull --ff-only upstream "refs/pull/$(cat merge.txt)/merge";
+            fi
+      - run:
+          name: Install the latest version of Poetry
+          command: |
+            curl -sSL https://install.python-poetry.org | python3 - --version 1.3.0
+            poetry --version
+      - run:
+          name: Set BASH_ENV
+          command: |
+            set -e
+            sudo apt update
+            sudo apt-get update
+            sudo apt install -qq graphviz optipng libxft2 graphviz-dev
+            echo "set -e" >> $BASH_ENV
+            echo "export OPENBLAS_NUM_THREADS=4" >> $BASH_ENV
+            echo "export XDG_RUNTIME_DIR=/tmp/runtime-circleci" >> $BASH_ENV
+            echo "export PATH=~/.local/bin/:$PATH" >> $BASH_ENV
+            echo "export DISPLAY=:99" >> $BASH_ENV
```

Comment:
Do we necessarily need this part? In particular, do we need graphviz?

Reply:
Good catch. Removing.